### PR TITLE
Added Kdenlive snap

### PR DESCRIPTION
**Pull request by j-b-m**
Kdenlive snap packaging

### CODE REVIEW

File: kdenlive/setpath.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Set cache folder to local path, dependent on snap version
+export XDG_CACHE_HOME=$SNAP_USER_DATA/.cache-$SNAP_VERSION
+mkdir -p $XDG_CACHE_HOME
+
+echo $SNAP
+cd $SNAP
+#echo "testing MLT --------------------\n"
+#melt color:red out=200 -consumer xgl
+echo "testing kdenlive --------------------\n"
+desktop-launch $SNAP/usr/bin/kdenlive
```
Comment (dholbach) [inline kdenlive/setpath.sh]: Did you try the `desktop-qt5` launcher? https://github.com/ubuntu/snappy-playpen/blob/master/2048/snapcraft.yaml would be a simple example of this.

**Comment by j-b-m** [comment]
On Monday, October 10, 2016 10:22:27 AM CEST, Daniel Holbach wrote:
> @dholbach commented on this pull request.

Hi,

> Great work, just a small question...
>
> In kdenlive/setpath.sh:

> +desktop-launch $SNAP/usr/bin/kdenlive


> Did you try the desktop-qt5 launcher? 
> https://github.com/ubuntu/snappy-playpen/blob/master/2048/snapcraft.yaml 
> would be a simple example of this.

Well, desktop-qt5 is not a launcher, it is a meta package that you can use 
in the after: of a part to make sure Qt5 related stuff is installed, and I 
use it in the snapcraft file. 

The example you pointed also use desktop-launch as a command (line 9).

So I am just waiting for this dbus stuff workaround and then I can get out 
of devmode...

Regards
jb

**Comment by dholbach** [comment]
The `desktop-qt5` cloud part, adds `stage-packages`, but also comes with a launcher, which adds a couple of tricks for integration: https://github.com/ubuntu/snapcraft-desktop-helpers/tree/master/qt

**Comment by j-b-m** [comment]
Oh maybe I misunderstood something then, sorry. I was under the impressions that desktop-qt5 stage package installed a Qt specific version of desktop-launch, called "desktop-launch". Looking at the desktop-helpers repositiory you linked, I cannot figure the name of a qt5 specific launcher? Can you give me a hint on how I should call that launcher ?

**Comment by dholbach** [comment]
https://github.com/ubuntu/snappy-playpen/blob/master/2048/snapcraft.yaml is an example of a very simple QML app.

**Comment by j-b-m** [comment]
Yes, but I am already doing exactly the same as the 2048 app:
I call "after: [desktop-qt5]" in snapcraft.yaml (in the mlt-snap part that is build before kdenlive part), and call "desktop-launch" from the setpath script used on application launch.

So, I don't understand what you suggest, sorry...

**Comment by dholbach** [comment]
I'm so sorry. I missed that you were already using it. Nevermind! Merging. Great work!

**Pull request merged**